apis/nucleus/src/components/listbox/ListBoxInline.jsx

```diff
@@ -161,7 +161,7 @@ export default function ListBoxInline({ options = {} }) {
   const showTitle = true;
 
   const searchVisible =
-    (search === true || ((search === 'toggle' || search === 'inSelection') && showSearch)) && !selectDisabled();
+    (search === true || ((search === 'toggle' || (search === 'inSelection' && !layout.title)) && showSearch)) && !selectDisabled();
```

Comment:
The same goes for when toolbar is disabled. Something like
```suggestion
    (search === true || ((search === 'toggle' || (search === 'inSelection' && (!layout.title || !toolbar))) && showSearch)) && !selectDisabled();
```

Reply:
Good to know! Updated!